src/elements/kw-share-detail/kw-share-detail.html

```diff
@@ -352,7 +362,14 @@
                         <h2>Loading</h2>
                     </div>
                 </div>
-                <div id="share-container"></div>
+                <div class="content">
+                    <template is="dom-if" if="[[share.featured]]">
```

Comment:
This dom-if might do a lot of DOM operations since it is on every share of a feed. Maybe switch to a `hidden` attribute. (I'm also not objective cause I like hidden better in general haha)